docs/ops/movement/Transpose_1.md

```diff
@@ -6,30 +6,28 @@
 
 **Short description**: *Transpose* operation reorders the input tensor dimensions.
 
-**Attributes**:
+**Detailed description**: *Transpose* operation reorders the input tensor dimensions. Source indexes and destination indexes are bound by the formula: 
 
-No attributes available.
+    output[i(order[0]), i(order[1]), ..., i(order[N-1])] = input[i(0), i(1), ..., i(N-1)], 
```

Comment:
As it is a formula I suggest to enclose it within \f[ \f].
```suggestion
   \f[
    output[i(order[0]), i(order[1]), ..., i(order[N-1])] = input[i(0), i(1), ..., i(N-1)]
   \f]
   
   where i(j) is within the range `[0, (input.shape[j]-1)]`
```

Reply:
Just thought it would be more readable as kind of pseudo code instead of math formula style.
But I will change it back with some space fixes.

Current:
https://docs.openvinotoolkit.org/2021.3/openvino_docs_ops_movement_Transpose_1.html  
![image](https://user-images.githubusercontent.com/20084299/114539992-53fa5500-9c55-11eb-9a8c-d12e7ee1d9ba.png)

vs

![image](https://user-images.githubusercontent.com/20084299/114541804-77260400-9c57-11eb-9762-b97f755785f3.png)

Reply:
I believe it looks better now
![image](https://user-images.githubusercontent.com/20084299/115342975-6cb1c000-a1ab-11eb-9c58-6a70ad2d28d8.png)